docs/installation/client-server.rst

```diff
@@ -0,0 +1,37 @@
+.. _install_ksql-client-server:
+
+Using KSQL in Client-Server Mode
```

Comment:
We don't want to call it different "modes".   See our offline discussion on how to structure the "Install" section.

Reply:
Filed this to work on immediately after merging: https://confluentinc.atlassian.net/browse/DOCS-344